lib/Horde/SessionHandler/Storage/File.php

```diff
@@ -129,14 +132,22 @@ public function write($id, $session_data)
     }
 
     /**
+     * @param string $id
+     * @return bool
      */
-    public function destroy($id)
+    public function destroy($id): bool
     {
         $this->close();
 
-        $filename = $this->_params['path'] . '/' . self::PREFIX . $id;
+        if (preg_match('/^[A-Za-z0-9,-]{22,256}$/', $id) !== 1) {
```

Comment:
if we use it in multiple places, maybe we should just add a method for that.

We could even add a protected method in the abstract `Horde_SessionHandler_Storage` class so all implementations can use it.